src/NHSD.BuyingCatalogue.Identity.Api/Views/Consent/Index.cshtml

```diff
@@ -31,6 +32,14 @@
         <div data-test-id="login-page">
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-full">
+                    <div data-test-id="go-back-section" class="nhsuk-back-link">
+                        <a data-test-id="go-back-link" class="nhsuk-back-link__link" href="/">
+                            <svg class="nhsuk-icon nhsuk-icon__chevron-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                                <path d="M8.5 12c0-.3.1-.5.3-.7l5-5c.4-.4 1-.4 1.4 0s.4 1 0 1.4L10.9 12l4.3 4.3c.4.4.4 1 0 1.4s-1 .4-1.4 0l-5-5c-.2-.2-.3-.4-.3-.7z"></path>
+                            </svg>
+                            Back to Public Browse
```

Comment:
are you sure about the wording here? it is not consistent with what we use elsewhere and also the user does not know it as 'Public Browse' - that is what we call it!

Reply:
No problem, what should it be called. E.g. the others are back to login? "Back to main page"?

Reply:
now that is a very good question ... let me have a root around in the fe apps and I will get back to you

Reply:
ok we have `Back` and `Go back to previous page` - might be one for Josh

Reply:
I'll send him a message

Reply:
According to Josh & Luke they want it to be named "Back to homepage", so I have updated the code